### DISCUSSION TITLE
Legacy Skin Packs

### DISCUSSION THREAD

**Discussion by Pigzone00** [RedRain0o0/legacy-skins#8] [(B) Legacy Console Edition Skin Pack Ports]
Legacy Skin Packs is a resource pack made for mods [Legacy4J](https://modrinth.com/mod/legacy4j) and [LegacySkins](https://modrinth.com/mod/legacy-skins) that aims to bring back all the Skin Packs from the Legacy Console Editions of Minecraft.

Edit from @Jab125:
Images:
![image](https://github.com/user-attachments/assets/c85d217c-d47a-4a07-8830-1c3b33e6e11c)

**Comment by JackxnIsntFunny** [comment]
where is the resource pack?

**Reply by AzaleaCatgirl99** [reply]
> where is the resource pack?

https://modrinth.com/resourcepack/legacy-skin-packs

**Comment by jakeyhanna** [comment]
How did you make the model appear in the menu??

When i try to make a pack with my cpmmodel the skin doesn't render but it works fine

**Reply by Pigzone00** [reply]
Here is an example pack which might help
[Example Skin Pack.zip](https://github.com/user-attachments/files/18038212/Example.Skin.Pack.zip)
